src/elements/Burger/BurgerMenu.tsx

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable @next/next/no-html-link-for-pages */
+/* eslint-disable react/forbid-component-props */
```

Comment:
`forbid-component-props` probably it's because you are trying to pass props that forbbiden, so you can try something like this: `<Link> =>  <a className={styles} /> => <Image />`